stress: failed test in cockroach/cli/cli.test: Example_zone

Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/b624f98deda333293ba82b5bf9bf603e6670728d

Stress build found a failed test:

```
=== RUN   Example_zone
I160801 05:19:19.804500 storage/engine/rocksdb.go:353  opening in memory rocksdb instance
W160801 05:19:19.807813 gossip/gossip.go:1006  not connected to cluster; use --join to specify a connected node
I160801 05:19:19.808906 server/node.go:397  store store=0:0 ([]=) not bootstrapped
I160801 05:19:19.811386 storage/replica_command.go:1716  store=1:1 ([]=) range=1 [/Min-/Max): new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 408341h19m24.310769663s following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0 [physicalTime=2016-08-01 05:19:19.811276504 +0000 UTC]
I160801 05:19:19.811965 server/node.go:323  **** cluster {b87a5076-42cf-47e6-a46a-ebe5575dd2f3} has been created
I160801 05:19:19.811998 server/node.go:324  **** add additional nodes by specifying --join=127.0.0.1:34395
I160801 05:19:19.812661 storage/store.go:2774  system config not yet available
I160801 05:19:19.812734 server/node.go:410  initialized store store=1:1 ([]=): {Capacity:536870912 Available:536870912 RangeCount:0}
I160801 05:19:19.812769 server/node.go:298  node ID 1 initialized
I160801 05:19:19.812825 storage/stores.go:302  read 0 node addresses from persistent storage
I160801 05:19:19.812884 server/node.go:533  connecting to gossip network to verify cluster ID...
I160801 05:19:19.813618 server/node.go:554  node connected via gossip and verified as part of cluster {"b87a5076-42cf-47e6-a46a-ebe5575dd2f3"}
I160801 05:19:19.813657 server/node.go:351  node=1: started with [[]=] engine(s) and attributes []
I160801 05:19:19.813686 server/server.go:427  starting https server at 127.0.0.1:45865
I160801 05:19:19.813705 server/server.go:428  starting grpc/postgres server at 127.0.0.1:34395
I160801 05:19:19.814040 storage/split_queue.go:100  splitting store=1:1 ([]=) range=1 [/Min-/Max) at keys [/Table/11/0 /Table/12/0 /Table/13/0 /Table/14/0]
I160801 05:19:19.814233 sql/event_log.go:103  Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:34395} Attrs:} ClusterID:{UUID:b87a5076-42cf-47e6-a46a-ebe5575dd2f3} StartedAt:1470028759813637011}
I160801 05:19:19.847024 storage/replica_command.go:2305  store=1:1 ([]=) range=1 [/Min-/Max): initiating a split of this range at key /Table/11
I160801 05:19:19.847888 sql/event_log.go:103  Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:34395} Attrs:} ClusterID:{UUID:b87a5076-42cf-47e6-a46a-ebe5575dd2f3} StartedAt:1470028759813637011}
I160801 05:19:19.850759 storage/log.go:84  Range Event: "split", range: 1, info: {"UpdatedDesc":{"range_id":1,"end_key":"kw==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":2,"start_key":"kw==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160801 05:19:19.855220 storage/replica_command.go:2305  store=1:1 ([]=) range=2 [/Table/11-/Max): initiating a split of this range at key /Table/12
I160801 05:19:19.856488 storage/log.go:84  Range Event: "split", range: 2, info: {"UpdatedDesc":{"range_id":2,"start_key":"kw==","end_key":"lA==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":3,"start_key":"lA==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160801 05:19:19.860077 storage/replica_command.go:2305  store=1:1 ([]=) range=3 [/Table/12-/Max): initiating a split of this range at key /Table/13
I160801 05:19:19.862502 storage/log.go:84  Range Event: "split", range: 3, info: {"UpdatedDesc":{"range_id":3,"start_key":"lA==","end_key":"lQ==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":4,"start_key":"lQ==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
I160801 05:19:19.865088 storage/replica_command.go:2305  store=1:1 ([]=) range=4 [/Table/13-/Max): initiating a split of this range at key /Table/14
I160801 05:19:19.866648 storage/log.go:84  Range Event: "split", range: 4, info: {"UpdatedDesc":{"range_id":4,"start_key":"lQ==","end_key":"lg==","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2},"NewDesc":{"range_id":5,"start_key":"lg==","end_key":"//8=","replicas":[{"node_id":1,"store_id":1,"replica_id":1}],"next_replica_id":2}}
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key ls]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key set system --file=./testdata/zone_attrs.yaml]
Error: error reading zone config: open ./testdata/zone_attrs.yaml: no such file or directory
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key ls]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key get system.nonexistent]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key get system.lease]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key set system --file=./testdata/zone_range_max_bytes.yaml]
Error: error reading zone config: open ./testdata/zone_range_max_bytes.yaml: no such file or directory
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key get system]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key rm system]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key ls]
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key rm .default]
Error: unable to remove .default
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key set .default --file=./testdata/zone_range_max_bytes.yaml]
Error: error reading zone config: open ./testdata/zone_range_max_bytes.yaml: no such file or directory
[zone --host=127.0.0.1 --port=34395 --insecure=false --ca-cert=/tmp/cli-test717586342/ca.crt --cert=/tmp/cli-test717586342/node.crt --key=/tmp/cli-test717586342/node.key get system]
--- FAIL: Example_zone (0.52s)
got:
zone ls
.default
zone set system --file=./testdata/zone_attrs.yaml
error reading zone config: open ./testdata/zone_attrs.yaml: no such file or directory
zone ls
.default
zone get system.nonexistent
system.nonexistent not found
zone get system.lease
.default
replicas:
- attrs: []
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 86400
zone set system --file=./testdata/zone_range_max_bytes.yaml
error reading zone config: open ./testdata/zone_range_max_bytes.yaml: no such file or directory
zone get system
.default
replicas:
- attrs: []
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 86400
zone rm system
DELETE 0
zone ls
.default
zone rm .default
unable to remove .default
zone set .default --file=./testdata/zone_range_max_bytes.yaml
error reading zone config: open ./testdata/zone_range_max_bytes.yaml: no such file or directory
zone get system
.default
replicas:
- attrs: []
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 86400
want:
zone ls
.default
zone set system --file=./testdata/zone_attrs.yaml
INSERT 1
replicas:
- attrs: [us-east-1a, ssd]
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 86400
zone ls
.default
system
zone get system.nonexistent
system.nonexistent not found
zone get system.lease
system
replicas:
- attrs: [us-east-1a, ssd]
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 86400
zone set system --file=./testdata/zone_range_max_bytes.yaml
UPDATE 1
replicas:
- attrs: [us-east-1a, ssd]
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 86400
zone get system
system
replicas:
- attrs: [us-east-1a, ssd]
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 86400
zone rm system
DELETE 1
zone ls
.default
zone rm .default
unable to remove .default
zone set .default --file=./testdata/zone_range_max_bytes.yaml
UPDATE 1
replicas:
- attrs: []
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 86400
zone get system
.default
replicas:
- attrs: []
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 86400
```

Run Details:

```
0 runs so far, 0 failures, over 5s
0 runs so far, 0 failures, over 10s
1 runs completed, 1 failures, over 13s
FAIL
```
Please assign, take a look and update the issue accordingly.

--- a/Makefile
+++ b/Makefile
@@ -162,13 +162,13 @@ coverage:
 # given all tests in the package will be run).
 .PHONY: stress
 stress:
-	$(GO) test -v $(GOFLAGS) -i -c $(PKG) -o stress.test
-	cd $(PKG) && stress $(STRESSFLAGS) $(CURRENTDIR)/stress.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
+	$(GO) test -v $(GOFLAGS) -i -c $(PKG) -o $(PKG)/stress.test
+	cd $(PKG) && stress $(STRESSFLAGS) ./stress.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
 
 .PHONY: stressrace
 stressrace:
-	$(GO) test $(GOFLAGS) -race -v -i -c $(PKG) -o stress.test
-	cd $(PKG) && stress $(STRESSFLAGS) $(CURRENTDIR)/stress.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
+	$(GO) test -v $(GOFLAGS) -i -c $(PKG) -o $(PKG)/stress.test -race
+	cd $(PKG) && stress $(STRESSFLAGS) ./stress.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
 
 .PHONY: acceptance
 acceptance:

